build.gradle

```diff
@@ -50,7 +50,7 @@ allprojects {
 		springVersionDefault = '5.3.0-SNAPSHOT'
 		springVersion = project.hasProperty('springVersion') ? getProperty('springVersion') : springVersionDefault
 		springRetryVersion = '1.2.5.RELEASE'
-		springAmqpVersion = '2.3.0.BUILD-SNAPSHOT' // TODO update to 2.3.0-SNAPSHOT when available
+		springAmqpVersion = '2.3.0-SNAPSHOT'
 		springDataCommonsVersion = '2.4.0-SNAPSHOT'
```

Comment:
I had to upgrade this version because my local build was failing

```
* What went wrong:
Execution failed for task ':spring-batch-infrastructure:compileJava'.
> Could not resolve all files for configuration ':spring-batch-infrastructure:compileClasspath'.
   > Could not find org.springframework.amqp:spring-amqp:2.2.7.
```